doc/src/manual/noteworthy-differences.md

```diff
@@ -197,14 +197,19 @@ For users coming to Julia from R, these are some noteworthy differences:
 
 ## Noteworthy differences from Python
 
-  * Julia requires `end` to end a block. Unlike Python, Julia has no `pass` keyword.
+  * Julia's `for`, `if`, `while`, etc. blocks are terminated by the `end` keyword. Indentation level
+    is not significant as it is in Python. Unlike Python, Julia has no `pass` keyword.
+  * Strings are denoted by double quotation marks (`"text"`) in Julia (with three double quotation marks for multi-line strings), whereas in Python they can be denoted either by single (`'text'`) or double quotation marks (`"text"`). Single quotation marks are used for characters in Julia (`'c'`).
+  * String concatenation is done with `*` in Julia, not `+` like in Python. Analogously, string repetition is done with `^`, not `*`.
```

Comment:
Maybe talk about "implicit concatenation" as in `"a" "b"` in Python does not exist in Julia?  (Maybe it's not the technical name.)

Reply:
I was not aware of this Python behavior until now (but probably it is not best practice to use it anyhow). I'll include this point.